exercises/practice/binary/.meta/config.json

```diff
@@ -1,6 +1,6 @@
 {
   "authors": [
-    "axtens"
+    "jalih"
```

Comment:
@axtens Maybe it makes sense here to have both you and jalih as authors? I think you've both contributed to the exercise, so to me that makes sense.

Reply:
Thank you.